routes/members.js

```diff
@@ -50,4 +52,36 @@ router.get('/', membersController.getMembers)
 
 router.get('/idle', membersController.getIdleMembers)
 
+/**
+ * @swagger
+ * /members/intro/:username:
+ *   post:
+ *     summary: Posts details of the recruiter
+ *     tags:
+ *       - Members
+ *     responses:
+ *       200:
+ *         description: Details of the recruiter and the member in which recruiter is interested
+ *         content:
+ *           application/json:
+ *             schema:
+ *               $ref: '#/components/schemas/recruiters'
+ *
+ *       404:
+ *         description: notFound
+ *         content:
+ *           application/json:
+ *             schema:
+ *               $ref: '#/components/schemas/errors/notFound'
+ *
+ *       500:
+ *         description: serverUnavailable
+ *         content:
+ *           application/json:
+ *             schema:
+ *               $ref: '#/components/schemas/errors/serverUnavailable'
+ */
+
+router.post('/intro/:username', recruiterValidator.validateRecruiter, recruitersController.addRecruiter)
```

Comment:
Why does the route need the `username` query param? 🤔
Also, is this route supposed to be consumed by anonymous users too?

Reply:
The form is linked with the members of RDS, that's the reason we need the `username` this is equivalent to RDS member username. 
Anonymous users( Recruiters: Genuine Case ) will trigger this route.